src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+
+type Props = {
+  todos: Todo[],
+  status: Status,
+  setStatus: (status: Status) => void,
+  deleteCompletedTodos: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  status,
+  setStatus,
+  deleteCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${todos.filter(todo => !todo.completed).length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={
+            classNames(
+              'filter__link',
+              {
+                selected: status === Status.ALL,
+              },
+            )
+          }
```

Comment:
you can change the line for better readability if you want
```suggestion
          className={classNames(
            'filter__link', { selected: status === Status.ALL },
          )}
```